dags/analyzer_helper/discord/utils/is_user_bot.py

```diff
@@ -0,0 +1,40 @@
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class UserBotChecker:
+    """
+    A class to check if a user is a bot in a specific platform.
+
+    Attributes:
+        platform_id (str): The ID of the platform.
+        client (MongoClient): The MongoDB client instance.
+        db (Database): The database instance for the platform.
+        guildmembers_collection (Collection): The collection of guild members.
+    """
+
+    def __init__(self, platform_id):
+        """
+        Initializes the UserBotChecker with the given platform ID.
+
+        Args:
+            platform_id (str): The ID of the platform.
+        """
+        self.client = MongoSingleton.get_instance().client
+        self.platform_id = platform_id
+        self.db = self.client[self.platform_id]
+        self.guildmembers_collection = self.db["guildmembers"]
```

Comment:
The `guildmembers` collection is under the `guild_id` database, So in our case the `UserBotChecker`'s input is guild_id instead of platform_id